src/libs/actions/CompanyCards.ts

```diff
@@ -740,6 +725,39 @@ function openPolicyCompanyCardsFeed(policyID: string, feed: CompanyCardFeed) {
     API.read(READ_COMMANDS.OPEN_POLICY_COMPANY_CARDS_FEED, parameters);
 }
 
+function flatAllCardsList(allCardsList: OnyxCollection<WorkspaceCardsList>, workspaceAccountID: number): Record<string, Card> | undefined {
+    if (!allCardsList) {
+        return;
+    }
+
+    let cards: Record<string, Card> = {};
+
+    Object.entries(allCardsList).forEach(([key, allCards]) => {
+        if (!key.includes(workspaceAccountID.toString()) || key.includes(CONST.EXPENSIFY_CARD.BANK)) {
+            return;
+        }
+
+        const {cardList, ...feedCards} = allCards ?? {};
+        cards = {...cards, ...feedCards};
+    });
+
+    return cards;
```

Comment:
```suggestion
return Object.entries(allCardsList).reduce((acc, [key, allCards]) => {
    if (!key.includes(workspaceAccountID.toString()) || key.includes(CONST.EXPENSIFY_CARD.BANK)) {
        return acc;
    }

    const {cardList, ...feedCards} = allCards ?? {};
    return {...acc, ...feedCards};
}, {});
```

super minor: can we do something like this?